packages/terra-dialog/src/Dialog.jsx

```diff
@@ -38,11 +38,13 @@ const Dialog = ({ header, footer, onClose, children, ...customProps }) => {
     'dialog',
   ]);
 
-  const actionHeader = <div className={cx('dialog-header')}><ActionHeader title={header} onClose={onClose} /></div>;
+  const closeButton = onClose ? <div className={cx('dialog-header-close')}><button className={cx('close')} onClick={onClose}><span className={cx('close-icon')} /></button></div> : null;
```

Comment:
Let's use the terra-button here with the utility variant.

Reply:
https://github.com/cerner/terra-core/pull/1250/commits/65b9979e103fb964030393be61f1e77f66aedb40